app/viewmodels/ImportVatViewModel.scala

```diff
@@ -0,0 +1,266 @@
+    "cf.account.vat.your-certificates.heading"
+  )
+
+  private def populateNotificationPanel(implicit messages: Messages, appConfig: AppConfig): HtmlFormat.Appendable = {
+    val htmlContent: Html = HtmlFormat.fill(
+      Seq(
+        linkComponent(
+          linkMessage = messages("cf.import-vat.requested-certificates-available-link-text"),
+          location = appConfig.requestedStatements(C79Certificate),
+          preLinkMessage = Some(messages("cf.account.detail.requested-certificates-available-text.pre")),
+          postLinkMessage = Some(messages("cf.account.detail.requested-certificates-available-text.post")),
+          pClass = "govuk-body govuk-!-margin-bottom-1"
+        )
+      )
+    )
+
+    divComponent(
+      content = htmlContent,
+      classes = Some("notifications-panel"),
+      id = Some("notification-panel")
+    )
+  }
+
+  private def populateCurrentStatements(
+    certsForAllEoris: Seq[VatCertificatesForEori]
+  )(implicit msgs: Messages): Seq[ImportVatCurrentStatementRow] =
+    certsForAllEoris.indices
+      .map { historyIndex =>
+        if (certsForAllEoris(historyIndex).currentCertificates.nonEmpty) {
+
+          val eoriHeading: Option[HtmlFormat.Appendable] = populateEoriHeading(certsForAllEoris, historyIndex)
+
+          val divContentRows: Seq[HtmlFormat.Appendable] = populateDivComponent(certsForAllEoris, historyIndex)
+
+          val dlComponentRowContent = dlComponent(
+            content = HtmlFormat.fill(divContentRows.map(htmlFormat => Html(htmlFormat.body))),
+            classes = Some("govuk-summary-list statement-list c79-statements"),
+            id = Some(s"statements-list-$historyIndex")
+          )
+
+          ImportVatCurrentStatementRow(eoriHeading = eoriHeading, dlComponentRow = dlComponentRowContent)
+        } else {
+          ImportVatCurrentStatementRow()
+        }
+      }
+      .filterNot(_.eoriHeading.contains(HtmlFormat.empty))
+
+  private def populateCurrentStatNotAvailableGuidance(implicit messages: Messages): HtmlFormat.Appendable =
+    pComponent("cf.account.vat.no-certificates-available", id = Some("no-certificates-available-text"))
+
+  private def populateCertsOlderThan6MonthsGuidance(serviceUnavailableUrl: Option[String])(implicit
+    messages: Messages
+  ): GuidanceRowWithParagraph =
+    GuidanceRowWithParagraph(
+      h2Heading = h2Component(
+        "cf.account.vat.older-certificates.heading",
+        id = Some("missing-certificates-guidance-heading"),
+        classes = "govuk-heading-m govuk-!-margin-top-9"
+      ),
+      link = Some(
+        linkComponent(
+          "cf.account.vat.older-certificates.description.link",
+          location = serviceUnavailableUrl.getOrElse(emptyString),
+          preLinkMessage = Some("cf.account.vat.older-certificates.description.1")
+        )
+      )
+    )
+
+  private def populateChiefDeclarationGuidance(implicit
```

Comment:
do we think its worth grouping the implicit keyword with the following part when its moved onto a new line? like we do on 105?